test/micro/org/openjdk/bench/java/lang/foreign/points/PointsAlloc.java

```diff
@@ -59,4 +62,21 @@ public Object panama_alloc() throws Throwable {
         return new PanamaPoint(0, 0);
     }
 
+    static final int NUM_CIRCLE_POINTS = 100;
+
+    @Benchmark
+    @OperationsPerInvocation(NUM_CIRCLE_POINTS)
+    public Object circle_by_value() {
+        try (Arena arena = Arena.ofConfined()) {
+            return Circle.byValue(arena, NUM_CIRCLE_POINTS);
+        }
+    }
+
+    @Benchmark
+    @OperationsPerInvocation(NUM_CIRCLE_POINTS)
+    public Object circle_by_ptr() {
+        try (Arena arena = Arena.ofConfined()) {
+            return Circle.byPtr(arena, NUM_CIRCLE_POINTS);
+        }
+    }
```

Comment:
I think it would be better to put the new benchmark in a separate benchmark class (e.g. a new variant of the `CallOverheadXXX` benchmark. There should already be some examples of benchmarks that pass structs), where the time unit is set to nanoseconds, so we can measure just the overhead of a single call.

Reply:
Done.